packages/notte-sdk/src/notte_sdk/endpoints/sessions.py

```
@@ -883,6 +957,115 @@ def cdp_url(self) -> str:
+            screenshot = await page.screenshot(path="screenshot.png")
+        ```
+
+        Returns:
+            PlaywrightPage: An async Playwright page instance connected to the session.
+
+        Raises:
+            ImportError: If playwright is not installed. Install with `pip install notte-sdk[playwright]`
+            ValueError: If the session hasn't been started yet (no session_id available).
+        """
+        if not _async_playwright_available:
+            raise ImportError("Playwright not installed. Use `pip install notte-sdk[playwright]` to install it.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)